docs/update-module.md

```diff
@@ -0,0 +1,104 @@
+# Updating a module
+
+When you have improvements or fixes to a module or want to address limitations, you can contribute updates. You fork the module you want to work with.
+
+?> **Tip:** If you're creating a module, see [Contributing to the IBM Cloud Terraform modules project](/contribute-module.md).
+
+## Overall process
+
+1.  Set up your development environment.
+1.  Fork the repo and clone it locally.
+1.  Make your changes.
+1.  Test your changes with example code.
+1.  Open a pull request.
+
+## Before you begin
+
+[inc-prereqs-dev.md](inc-prereqs-dev.md ':include')
+
+## Fork, clone, and initialize the repo
+
+Fork the module repo that you want to update.
+
+1.  Navigate to the repository that you want to update.
+1.  Fork the repo by clicking **Fork** in the top-right corner of the page.
+
+    - Clear the **Copy the main branch only** option to copy all branches to your fork.
+    - If you plan to create another module from this code, type a new repository name.
+1.  Clone your fork:
+
+    - To clone by using SSH, run the following command. You need a valid SSH key for github.com. For more information, see the steps in the [GitHub docs](https://docs.github.com/en/authentication/connecting-to-github-with-ssh).
+
+        ```bash
+        gh clone git@github.com:<REPO_OWNER>/<REPO_NAME>.git
+        ```
+
+    - To clone by using HTTPS, run the following command.
+
+        ```bash
+        git clone https://github.ibm.com/<REPO_OWNER>/<REPO_NAME>.git
+        ```
+
+    Where `<REPO_OWNER>` is the owner of the forked repository and `<REPO_NAME>` is the name of the repo you forked.
+1.  Confirm the remote repository for your fork.
+
+    1.  Locally, list the remote repo:
+
+        ```bash
+        git remote -v
+        ```
+
+    1.  If necessary, add or update the upstream repo. For example, if you use SSH, run this command:
+
+        ```bash
+        git remote add upstream git@github.com:terraform-ibm-modules/<REPO_NAME>.git
+        ```
```

Comment:
Optionally create a branch, it is not required but I would always keep work in a branch and try to keep my main fork sync'd with the upstream main.

Reply:
Daniel, I've got that step in the section about [updating the terraform files](https://terraform-ibm-modules.github.io/documentation/#/contribute-module?id=update-the-terraform-files).

> From the root of the newly cloned repo, create a branch to work in by running the command git checkout -b ＜YOUR_WORKING_BRANCH>. Replace ＜YOUR_WORKING_BRANCH> with your own branch name.